.github/workflows/compute.yml

```diff
@@ -34,8 +34,23 @@ jobs:
         with:
           node-version: "20"
 
-      - name: Install Bun
-        run: npm install -g bun
+      - uses: oven-sh/setup-bun@v1
+
+      - name: Generate Supabase Types
+        run: |
+          yarn install
+          yarn run "supabase:generate:remote"
+        env:
+          SUPABASE_PROJECT_ID: ${{ secrets.SUPABASE_PROJECT_ID }}
+          SUPABASE_ACCESS_TOKEN: ${{ secrets.SUPABASE_ACCESS_TOKEN }}
+
+      - name: Generate Supabase Types
+        run: |
+          yarn install
+          yarn run "supabase:generate:remote"
+        env:
+          SUPABASE_PROJECT_ID: ${{ secrets.SUPABASE_PROJECT_ID }}
+          SUPABASE_ACCESS_TOKEN: ${{ secrets.SUPABASE_ACCESS_TOKEN }}
```

Comment:
Why do we need supabase for setting prices to labels. This should be removed.

Reply:
This is used by the command that gives permissions to users for setting certain labels.